src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -735,24 +735,41 @@ void lsf_driver_kill_job(void *_driver, void *_job) {
         char **argv = (char **)calloc(2, sizeof *argv);
         CHECK_ALLOC(argv);
         argv[0] = driver->remote_lsf_server;
-        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGKILL",
+        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGTERM",
                            job->lsf_jobnr_char);
-
-        spawn_blocking(driver->rsh_cmd, 2, (const char **)argv, NULL, NULL);
-
         free(argv[1]);
         free(argv);
+
+        char **argv2 = (char **)calloc(2, sizeof *argv2);
+        argv2[0] = driver->remote_lsf_server;
```

Comment:
I have not checked what CHECK_ALLOC is doing, but should it be included here as well?

Reply:
`CHECK_ALLOC` was something that was introduced because that's what `util_malloc` did. There is no reason for it to exist.